ui/src/app/edge/live/digitalinput/digitalinput.component.html

```diff
@@ -0,0 +1,19 @@
+<ng-container *ngIf="service.currentEdge | async as edge">
+    <ng-container *ngIf="(edge.currentData | async)['channel'] as currentData">
+        <ion-card button (click)="presentModal()">
+            <ion-item color="light" lines="full">
+                <ion-icon slot="start" size="large" name="radio-button-off-outline"></ion-icon>
+                <ion-label translate>General.digitalInputs</ion-label>
+            </ion-item>
+
+            <ion-card-content>
+                <table class="full_width">
+                    <tr>
+                        <td style="width:65%" translate>Anzahl Komponenten</td>
```

Comment:
"translate" needs an identifier for the string, right? I don't think it will work this way.